Ash.FrontEnd/src/style.less

```diff
@@ -1,55 +1,59 @@
+@import '~normalize.css';
+
 // Application wide styles go here
 @import '~variables.less';
 
 html {
-  height: 100%;
   font-family: 'lato';
+  -webkit-box-sizing: border-box;
```

Comment:
If we get a post-processor it'll take care of all the -webkit and -moz stuff for us

Reply:
Hmm that would be nice.  I don't know much about post processors, though, so maybe you can help me with that?